molgenis-ui-settings/src/main/frontend/src/store/getters.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import { EntityToStateMapper } from '@molgenis/molgenis-ui-form'
+
+const getters = {
+  getMappedFields: state => {
+    return state.rawSettings &&
+      state.rawSettings.meta &&
+      EntityToStateMapper.generateFormFields(state.rawSettings.meta)
```

Comment:
why have the raw settings and transform on get ? is this to store the state ? is this performant when building 'larger' forms ? I suggest we discusses this setup outside of the scope of the pull request